tests/controllers/index.ts

```diff
@@ -0,0 +1,15 @@
+import Container from 'typedi';
+import FeedbackService from '../../services/FeedbackService';
+import { FeedbackController } from '../../api/controllers/FeedbackController';
+
+export class ControllerFactory {
+  private static feedbackController: FeedbackController = null;
+
+  public static feedback(): FeedbackController {
+    if (!ControllerFactory.feedbackController) {
+      const feedbackService = Container.get(FeedbackService);
```

Comment:
Can we avoid interacting with the container entirely and manually construct the feedback service?